src/Identities.ts

```diff
@@ -0,0 +1,54 @@
+import { Context, Identity, registerIdentity, RegisterIdentityParams } from '~/internal';
+import { ProcedureMethod } from '~/types';
+import { createProcedureMethod } from '~/utils/internal';
+
+/**
+ * Handles all Identity related functionality
+ */
+export class Identities {
```

Comment:
Shouldn't `getCurrentIdentity` be in this namespace as well?

Reply:
To me the idea of "currentIdentity" has to do more with SDK configuration than with interacting with Identities. 

In a sense the corresponding "setter" is the `setSigner` method and I think they should be kept together. The usage of the "currentIdentity" is different than all of the other methods here imo.

That said I seem to be in the minority here so I can move it if you still think it belong in the namespace.

Reply:
You make a very good point. Let's leave it as is for now